gen_judgment.py

```
@@ -41,17 +41,26 @@ def pairwise_judgment(question, baseline, answer, reference, configs, settings):
     if reference:
         prompt_args[f"REFERENCE"] = reference["messages"][-1]["content"]['answer']
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)